tests/utils/test_serializers.py

```diff
@@ -0,0 +1,63 @@
+from typing import Union
+
+import numpy as np
+import pytest
+import torch
+from pydantic import BaseModel, ConfigDict, PlainSerializer, PlainValidator
+from typing_extensions import Annotated
+
+from careamics.utils.serializers import array_to_json, list_to_numpy, list_to_torch
+
+Array = Annotated[
```

Comment:
It would be better to use the actual type that is in the code base (that way we are testing it!).